wiki/Music_theory/Time_signature/en.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - music theory
+  - mapping
+  - beatmapping
+  - tips
```

Comment:
these tags have little relation to the article (and "music theory" is too general imo)

Reply:
i didn't write the article but i agree lol